public/403.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en" >
+<head>
+  <meta charset="UTF-8">
+  <title>ERROR 403</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <div id="mainC">
+  <div class="message">
+    <h1>403</h1>
+    <h3>Forbidden | The Browser you're currently using does not supports <strong>susi.ai</strong></h3>
+  </div>
+  <div class="footer">
+    <a href="https://chat.susi.ai/" title="Home" target="_blank">SUSI.AI<span></span></a>
+    <p class="legal">copyright &copy; 2016&nbsp;-&nbsp;2017 FOSSASIA&trade; All rights reserved</p>
```

Comment:
![screen shot 2018-01-07 at 1 52 09 pm](https://user-images.githubusercontent.com/26062692/34647784-54a81c02-f3b2-11e7-8f52-9dd18a70caa4.png)
Also, it would be better if you changed the copyright part to the way it has been done on https://fossasia.org/. Thanks for the PR, though 😄

Reply:
Done @Remorax